opentok/opentok.py

```diff
@@ -311,7 +311,7 @@ def archive_url(self, archive_id=None):
             url = url + '/' + archive_id
         return url
 
-    def start_archive(self, session_id, has_audio=True, has_video=True, name=None, output_mode=OutputModes.composed):
+    def start_archive(self, session_id, has_audio=True, has_video=True, name=None, output_mode=OutputModes.composed,resolution="640x480"):
```

Comment:
Below (and from opentok docs) it says 
> If you set this 
          parameter and set the output_mode parameter to OutputModes.individual, the call to the 
          start_archive() method results in an error.`

However, we're setting a default value, and unconditionally adding to the payload.
This sounds like it will break individual mode.

I think maybe the resolution should only be added to payload if individual mode is not selected.

Reply:
@maikthomas Thanks for bringing this up. If that is the case, we need to take care of this condition carefully.

Reply:
I agree if the response body contains that property.